### ISSUE TITLE
Importing duplicate records does not update founded duplicates

### ISSUE THREAD

**Issue by primozn** [salesagility/SuiteCRM#2081]
#### Issue

When I attempt to re-import and updated Accounts export, I tell the system to import new and update existing. I then go through the process, and make sure to tell it to match based on e-mail (which are all 100% the same). None of the information was updated.
#### Expected Behavior

It should update changed fields.
#### Actual Behavior

Won't update. Only show duplicate records.
#### Steps to Reproduce

http://screencast.com/t/My7KRYfTK
#### Context

high priority
#### Your Environment
- SuiteCRM Version used: 7.7.3
- Browser name and version: Chrome 52.0.2743.116 m
- Environment name and version: MySQL, PHP 7.0.5
- Operating System and version: Windows NT WIN7-SRV 6.1 build 7601 (Windows 7 Ultimate Edition Service Pack 1) i586

**Comment by ijdavie** [comment]
Hi, @primozn, 

Importing updates to existing records in SuiteCRM will only update records where there is a matching ID in the import file this is how it is designed, The duplicate check on other fields will only return a list of duplicate records and will not update these. 

I will raise this as a suggestion.

**Comment by primozn** [comment]
If i understand you right with '**matching ID**' you mean '**Name**' field in case you want to update '**Accounts**'?

So, in **fields to check** i have to put only '**Name**' or nothing at all?
http://screencast.com/t/or4euZhBi

**Comment by murdal** [comment]
Hi @primozn - By matching ID @ijdavie means the GUID of the SuiteCRM record.

**Comment by virtualme123** [comment]
Ok I am doing this to account records and expecting a match on ID but when it imports I get an Error:
`Error ID already exists in this table `
So it matched the ID but didn't import it - seems strange?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Importing duplicate records does not update founded duplicates](https://trello.com/c/rlDfxgoK/79-importing-duplicate-records-does-not-update-founded-duplicates)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
